spec/integration/util/windows/security_spec.rb

```diff
@@ -387,9 +394,13 @@ def grant_everyone_full_access(path)
         end
 
         describe "#owner=" do
+          it "should accept the guest sid" do
+            expect(winsec.get_owner(path)).to eq(sids[:guest])
+          end
+
           it "should accept a user sid" do
-            winsec.set_owner(sids[:admin], path)
-            expect(winsec.get_owner(path)).to eq(sids[:admin])
+            winsec.set_owner(sids[:current_user], path)
+            expect(winsec.get_owner(path)).to eq(sids[:current_user])
```

Comment:
I'm a little concerned that if `set_owner` fails to perform the operation, and doesn't raise an exception, then the test will pass. Can we instead set the owner to `Guest`?